x-pack/plugins/task_manager/server/saved_objects/index.ts

```diff
@@ -72,5 +73,6 @@ export function setupSavedObjects(
         },
       } as estypes.QueryDslQueryContainer;
     },
+    modelVersions: taskModelVersions,
```

Comment:
@pgayvallet These changes to the task manager model version were added after you approved the PR (for `serverless.yml` changes only). Could you take a look at these changes as well? Thanks!